skunkworks_crow/src/main/res/layout/activity_web_view.xml

```diff
@@ -1,32 +1,30 @@
-<RelativeLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:layout_marginBottom="0dp"
     android:layout_marginLeft="0dp"
-    android:layout_marginRight="0dp"
     android:layout_marginTop="0dp"
+    android:layout_marginRight="0dp"
+    android:layout_marginBottom="0dp"
     android:orientation="vertical"
     android:padding="0dp">
 
-    <android.support.v7.widget.Toolbar
-        xmlns:android="http://schemas.android.com/apk/res/android"
+    <androidx.appcompat.widget.Toolbar xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The namespace `android` is already defined in the root element. This one can be removed

Reply:
I didn't add that, anyway that is no need

Reply:
I must have missed it while it was added in one of the earlier PRs. Apologies. And thanks for fixing it